apps/server/src/shared/infra/collaborative-storage/collaborative-storage.adapter.spec.ts

```diff
@@ -59,4 +61,11 @@ describe('CollaborativeStorage Adapter', () => {
 			);
 		});
 	});
+
+	describe('Delete Team from Nextcloud', () => {
+		it('should call the strategy', () => {
+			const teamIdMock = 'teamIdMock';
+			adapter.deleteTeam(teamIdMock);
```

Comment:
there seems to be an expect missing here.

I strongly suggest making sure you have seen each test failing at least once, to avoid false positives.
The easiest (and most fun) way to achieve that is by writing the test before you write the code that makes it pass